src/com/itacademy/eshop/product/Clothing.java

```diff
@@ -0,0 +1,21 @@
+package com.itacademy.eshop.product;
+
+import com.itacademy.eshop.product.types.Category;
+
+public class Clothing extends Product {
+
+
+    public Clothing(String name, double price, Category category) {
+        super(name, price, category);
+    }
+
+    @Override
+    public void displayProductInfo() {
+        System.out.println("This is a clothe");
```

Comment:
this comment can be ignored
grammatically, clothes dont have a singular form, so in this case you should still write "this is clothing" or "these are clothes"